libmalcontent/tests/app-filter.c

```diff
@@ -649,9 +655,17 @@ async_result_cb (GObject      *obj,
  * given in #GetAppFilterData.properties if queried for a UID matching
  * #GetAppFilterData.expected_uid. Intended to be used for writing ‘successful’
  * mct_manager_get_app_filter() tests returning a variety of values. */
+typedef enum
+{
+  /* Copied from accounts-service: */
```

Comment:
:(

Reply:
`/usr/include/accountsservice-1.0/act/act-user.h`

```
typedef enum {
        ACT_USER_ACCOUNT_TYPE_STANDARD,
        ACT_USER_ACCOUNT_TYPE_ADMINISTRATOR,
} ActUserAccountType;
```

Reply:
I think I might have copied that enum when originally writing this patch because it seemed less likely to cause future rebase conflicts than adding a dependency into the `meson.build` for this directory. Arguable though. I’d prefer for it to be fixed in a follow-up MR rather than here.